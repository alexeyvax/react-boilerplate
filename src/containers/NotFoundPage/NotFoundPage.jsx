import React from 'react';
import { Link } from 'react-router-dom';

const NotFoundPage = () => (
  <React.Fragment>
    <h2>This is a Not Found Page 404!</h2>
    <Link to="/">Go to main page</Link> {/* eslint-disable-line jsx-a11y/anchor-is-valid */}
  </React.Fragment>
);

export default NotFoundPage;
