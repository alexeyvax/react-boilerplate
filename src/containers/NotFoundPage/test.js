import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';

import NotFoundPage from './NotFoundPage';

it('renders correctly', () => {
  const tree = renderer
    .create((
      <MemoryRouter>
        <NotFoundPage />
      </MemoryRouter>
    ))
    .toJSON();

  expect(tree).toMatchSnapshot();
});
