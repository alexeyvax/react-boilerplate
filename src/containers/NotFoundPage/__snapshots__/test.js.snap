// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <h2>
    This is a Not Found Page 404!
  </h2>,
  <a
    href="/"
    onClick={[Function]}
  >
    Go to main page
  </a>,
  " ",
]
`;
